contracts/upgradeMocks/distribution/ZNSEthRegistrarMock.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+import { ZNSEthRegistrar } from "../../distribution/ZNSEthRegistrar.sol";
+import { Mock } from "../Mock.sol";
+
+ /* solhint-disable */
+contract ZNSEthRegistrarMock is ZNSEthRegistrar, Mock {}
```

Comment:
same here. let's add "upgrade" to the name so that it's clear why these mocks exist. for all the below contracts as well